src/typesGithub.ts

```diff
@@ -145,22 +157,31 @@ export interface Owner {
 export interface Subject {
   title: string;
   url: string | null;
-  state?: StateType;
+  state?: StateType; // This is not in the GitHub API, but we add it to the type to make it easier to work with
```

Comment:
Thanks for adding the comment! Gives some ideas about what we could improve in future, too -- e.g. wrapping subject into our own type.

Reply:
I was thinking of the same.  Ideally, I'd like to refactor to use #823 and it's ootb types for GitHub, and then have Gitify types that are narrowed down to the handful of fields needed to _actually_ power the experience (eg: title, color, icon, type, state, timestamp, url) rather than the full raw GitHub response types

Reply:
https://github.com/gitify-app/gitify/issues/828 for future us